ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/IdentityProviderConfigServiceIT.scala

```diff
@@ -0,0 +1,489 @@
+  )(implicit ec => { case Participants(Participant(ledger)) =>
+    def createAndCheck(
+        problem: String,
+        expectedErrorCode: ErrorCode,
+        identityProviderId: String = UUID.randomUUID().toString,
+        isDeactivated: Boolean = false,
+        issuer: String = UUID.randomUUID().toString,
+        jwksUrl: String = "http://daml.com/jwks.json",
+    ): Future[Unit] = {
+      for {
+        throwable <- ledger
+          .createIdentityProviderConfig(
+            identityProviderId,
+            isDeactivated,
+            issuer,
+            jwksUrl,
+          )
+          .mustFail(context = problem)
+      } yield assertGrpcError(
+        t = throwable,
+        errorCode = expectedErrorCode,
+        exceptionMessageSubstring = None,
+      )
+    }
+
+    for {
+      _ <- createAndCheck(
+        "empty identity_provider_id",
+        LedgerApiErrors.RequestValidation.InvalidField,
+        identityProviderId = "",
+      )
+      _ <- createAndCheck(
+        "invalid identity_provider_id",
+        LedgerApiErrors.RequestValidation.InvalidField,
+        identityProviderId = "!@",
+      )
+      _ <- createAndCheck(
+        "empty issuer",
+        LedgerApiErrors.RequestValidation.MissingField,
+        issuer = "",
+      )
+      _ <- createAndCheck(
+        "empty jwks_url",
+        LedgerApiErrors.RequestValidation.InvalidField,
+        jwksUrl = "",
+      )
+      _ <- createAndCheck(
+        "non valid url",
+        LedgerApiErrors.RequestValidation.InvalidField,
+        jwksUrl = "url.com",
+      )
+      createEmptyIdpConfig <- ledger
+        .createIdentityProviderConfig(CreateIdentityProviderConfigRequest(None))
+        .mustFail(context = "empty identity_provider_config")
+    } yield {
+      assertGrpcError(
+        t = createEmptyIdpConfig,
+        errorCode = LedgerApiErrors.RequestValidation.MissingField,
+        exceptionMessageSubstring = None,
+      )
+    }
+  })
+
+  test(
+    "IdentityProviderGetConfigInvalidArguments",
+    "Test argument validation for IdentityProviderConfigService#GetIdentityProviderConfig",
+    allocate(NoParties),
+    enabled = _.userManagement.supported,
+    disabledReason = "requires user management feature",
+  )(implicit ec => { case Participants(Participant(ledger)) =>
+    def createAndCheck(
+        problem: String,
+        expectedErrorCode: ErrorCode,
+        identityProviderId: String,
+    ): Future[Unit] = {
+      for {
+        throwable <- ledger
+          .getIdentityProviderConfig(
+            GetIdentityProviderConfigRequest(identityProviderId)
+          )
+          .mustFail(context = problem)
+      } yield assertGrpcError(
+        t = throwable,
+        errorCode = expectedErrorCode,
+        exceptionMessageSubstring = None,
+      )
+    }
+
+    for {
+      _ <- createAndCheck(
+        "empty identity_provider_id",
+        LedgerApiErrors.RequestValidation.InvalidField,
```

Comment:
Should not this be a `MissingField` as the above:
https://github.com/digital-asset/daml/pull/16230/files#diff-04d0b046b6564c9fabfa57ceda8148b1904da373953f61164ff2081abb7caabbR70
?